modules/storage-api/src/main/java/org/apache/ignite/internal/storage/util/StorageUtils.java

```diff
@@ -155,6 +155,15 @@ public static void throwStorageExceptionIfItCause(IgniteInternalCheckedException
         }
     }
 
+    /**
+     * Throws an {@link StorageException} if the storage already exists or is being created.
+     *
+     * @param storageInfo Storage information, for example in the format "table=user, partitionId=1".
+     */
+    public static void throwExceptionIfStorageExists(String storageInfo) {
+        throw new StorageException("Storage already exists or is being created: " + storageInfo);
```

Comment:
the method name says "ifStorageExists" but it throws unconditionally

Reply:
This method is not needed, I will remove it.